src/HeadProvider.js

```diff
@@ -66,6 +66,14 @@ export default class HeadProvider extends React.Component {
     },
   };
 
+  componentDidUpdate() {
```

Comment:
Curious, why in this lifecycle method versus, say, componentDidMount ?

Reply:
Just mixed up